ExampleApp/Objective-C/CustomCreditCardFormViewController.m

```diff
@@ -0,0 +1,138 @@
+#import "CustomCreditCardFormViewController.h"
+#import <OmiseSDK/OmiseSDK.h>
+
+@interface CustomCreditCardFormViewController ()
+
+@property (strong, nonatomic) IBOutlet OMSCardNumberTextField *cardNumberField;
+@property (strong, nonatomic) IBOutlet OMSCardNameTextField *cardNameField;
+@property (strong, nonatomic) IBOutlet OMSCardExpiryDateTextField *cardExpiryField;
+@property (strong, nonatomic) IBOutlet OMSCardCVVTextField *cardCVVField;
+
+@property (strong, nonatomic) IBOutlet UIBarButtonItem *doneButton;
+
+@property (strong) OMSSDKClient *omiseClient;
+
+@end
+
+@implementation CustomCreditCardFormViewController
+
+- (void)loadView {
+    [super loadView];
+    
+    if (!self.storyboard) {
+        self.view.backgroundColor = UIColor.whiteColor;
+        
+        self.cardNumberField = [[OMSCardNumberTextField alloc] init];
+        self.cardNumberField.translatesAutoresizingMaskIntoConstraints = false;
+        self.cardNumberField.placeholder = @"1234567812345678";
+        self.cardNameField = [[OMSCardNameTextField alloc] init];
+        self.cardNameField.translatesAutoresizingMaskIntoConstraints = false;
+        self.cardNameField.placeholder = @"John Appleseed";
+        self.cardExpiryField = [[OMSCardExpiryDateTextField alloc] init];
+        self.cardExpiryField.translatesAutoresizingMaskIntoConstraints = false;
+        self.cardExpiryField.placeholder = @"mm/YY";
```

Comment:
It should be MM/YY ?

Reply:
I just want to show that you can set a place holder here. But year this shouldn't be `mm` though

Reply:
Okie